src/structures/StageChannel.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const BaseGuildVoiceChannel = require('./BaseGuildVoiceChannel');
+
+/**
+ * Represents a guild stage channel on Discord.
+ * @extends {BaseGuildVoiceChannel}
+ */
+class StageChannel extends BaseGuildVoiceChannel {
+  _patch(data) {
+    super._patch(data);
+
+    /**
+     * The topic of the stage channel
+     * @type {?string}
+     */
+    this.topic = data.topic;
+  }
+
+  /**
+   * Sets the RTC region of the channel.
```

Comment:
Is this needed? it inherits from the base one 👀

Reply:
Yep, it's needed to specify a different return type.